homework-4/Program.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ConsoleApp
+{
+    public class Program
+    {
+        static List<int> list = new List<int>();
+        static Dictionary<int, string> dictionary = new Dictionary<int, string>();
+
+        static void Main(string[] args)
+        {
+            Menu();
+        }
+
+        static void Menu()
+        {
+            Console.WriteLine("Please enter a number from the given menu:");
+            Console.WriteLine("1. Add an element to the List.");
+            Console.WriteLine("2. Print all the elements present in the List.");
+            Console.WriteLine("3. Delete the last element from the List.");
+            Console.WriteLine("4. Delete the first element from the List.");
+            Console.WriteLine("5. Delete the middle element from the List.");
+            Console.WriteLine("6. Calculate the average of the elements present in the List.");
+            Console.WriteLine("7. Exit the application.");
+
+            string choice = Console.ReadLine();
+
+            switch (choice)
+            {
+                case "1":
+                    Console.Write("Enter a number to add to the List: ");
+                    int numToAdd = Convert.ToInt32(Console.ReadLine());

```

Comment:
what if i enter 'a' in the console will it be converted? Or will it throw an error. Also did you looked into int.TryParse()?